google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -1213,6 +1223,7 @@ public void close(@Nullable String message) {
       if (stream != null) {
         stream.close(message);
         span.end(TraceUtil.END_SPAN_OPTIONS);
+        OpenTelemetryTraceUtil.endSpan(openTelemetrySpan);
```

Comment:
Same also for end as for the other span method: Could we combine these in a util method?